### DISCUSSION TITLE
Enhance search with wildcard? Trying to identify mismatches

### DISCUSSION THREAD

**Discussion by nottheend** [firefly-iii/firefly-iii#8822] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/explanation/more-information/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

- I am frequently struggling with mismatches of bank statements and my recordings in firefly-iii
- I would like to use the `amount` search with a wildcard, e.g. `*`
- Use Case to cover: trying to find a mismatch of the bank statement and firefly-iii. Sometimes the only hint are the cents, i.e. the smallest unit of the currency which gives a hint

### Solution

`amount:*.84 `
This would search for any amount ending with `.84,` e.g. `4.84 `or `1044.84`

**Comment by JC5** [comment]
Hey, thanks for the suggestion!

You can use `less:1.23` and `more:1.23` if you want. A wildcard function will not be added I'm afraid.

**Comment by nottheend** [comment]
Thanks. The intention of this is to identify easier where I have a mismatch with the bank statement. Mismatches still happen way too often and I am frequently annoyed.
